sources/web/datalab/polymer/modules/GapiManager.ts

```diff
@@ -104,6 +104,19 @@ class GapiManager {
     return GapiManager._loadBigQuery().then(() => gapi.client.bigquery.tables.list(request));
   }
 
+  /**
+   * Fetches table details from BigQuery
+   */
+  public static getTableDetails(projectId: string, datasetId: string, tableId: string):
```

Comment:
I don't know if we want to assume the only tables we have come from BigQuery. How about we call this getBigQueryTable or getBigQueryTableDetails?